packages/extension-browser/src/devtools/views/controls/feedback-link.tsx

```diff
@@ -8,7 +8,7 @@ import ExternalLink from './external-link';
 import { evaluate } from '../../utils/inject';
 import { getCategories } from '../../utils/categories';
 import escapeRegExp = require('lodash/escapeRegExp');
-import { getCategoryName } from '@hint/utils/dist/src/i18n';
+import { getCategoryName } from '@hint/utils-i18n/dist/src/get-category-name';
```

Comment:
Yes, we may still have to point to `dist/src` in some cases, though I suspect with the smaller utils a number of these will be able to become direct references as all of their code will already be part of the browser extension. 

I'm willing to raise the ceiling by a few KB to keep this simpler too. The main goal is to avoid pulling in larger packages that we don't need.

Reply:
I can do some tests at the end to see if the size difference is acceptable changing the references to some of the new packages.